.github/workflows/bridge.yml

```
@@ -33,4 +33,14 @@ jobs:
           cp target/release/witnet-centralized-ethereum-bridge ./
 
       - name: Build docker Image
-        run: docker build -t witnet/witnet-centralized-ethereum-bridge:"${{ env.IMAGE_VERSION }}" -f ./docker/bridge/Dockerfile . && docker image ls 
\ No newline at end of file
+        run: docker build -t witnet/witnet-centralized-ethereum-bridge:"${{ env.IMAGE_VERSION }}" -f ./docker/bridge/Dockerfile . && docker image ls 
+
+      - name: Login to Docker hub Registry
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Push Image
+        run: |
+          docker push witnet/witnet-centralized-ethereum-bridge:"${{ env.IMAGE_VERSION }}
```

Is it really OK for this double quote to never be closed? :thinking: 
```suggestion
          docker push witnet/witnet-centralized-ethereum-bridge:"${{ env.IMAGE_VERSION }}"
```

---

no it should most definitely be closed, great catch 😂